lib/DidKeyDriver.js

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {LDKeyPair} = require('crypto-ld');
+const base58 = require('bs58');
```

Comment:
Note to self: figure out the difference between the `bs58` library and `forge`'s base58 export. (Both seem to be getting used interchangeably in our code?)